locale/ko/download/package-manager.md

```diff
@@ -37,7 +37,7 @@ title: 패키지 매니저로 Node.js 설치하기
 
 * [Android](#android)
 * [Arch Linux](#arch-linux)
-* [Debian and Ubuntu based Linux distributions, Enterprise Linux/Fedora and Snap packages](#debian-and-ubuntu-based-linux-distributions-enterprise-linux-fedora-and-snap-packages)
+* [데비안과 우분투 기반 리눅스 배포판. 엔터프라이즈 리눅스/페도라와 Snap 패키지](#debian-and-ubuntu-based-linux-distributions-enterprise-linux-fedora-and-snap-packages)
```

Comment:
The reason is that if you use "##" to generate an anchor and there's something out of 26 English alphabets, it will be ignored. And I've submitted this fixture you can have a review for that, this can keep what it is in the English version.

See：#2028. Hope it helps you :)